bigquery/indexer.py

```diff
@@ -207,7 +207,7 @@ def _create_table_from_view(bq_client, view):
 
 def index_table(es, bq_client, storage_client, index_name, table,
                 participant_id_column, sample_id_column, sample_file_columns,
-                deploy_project_id):
+                deploy_project_id, all_table_documents, all_sample_documents):
```

Comment:
This code works for 1000 Genomes where each participant document has exactly one sample document. But it doesn't work for encode, where a partcipant document can have more than one sample document. Eg see `kubectl exec -it es-data-0 curl localhost:9200/encode/type/ENCDO000AAD?pretty`


You can test encode with my config files. (But change the project in deploy.json to your project -- or don't deploy on GKE, just run locally.)